client/webserver/site/src/html/settings.tmpl

```diff
@@ -43,7 +43,8 @@
         servers.
         </p>
         <button id="addADex" class="bg2 selected">Add a DEX</button>
-        <button id="importAccount" class="bg2 selected">Import Account</button>
+        <button id="importAccount" class="bg2 selected ml-2">Import Account</button>
+        <button id="changeAppPW" class="bg2 selected ml-2">Change App Password</button>
```

Comment:
Let's make a new section for this button. This section is for DEX accounts.

Reply:
![image](https://user-images.githubusercontent.com/10324528/112408133-eaef8380-8d1f-11eb-87d3-bd6f37ca34f0.png)